app/controllers/dashboard_controller.rb

```diff
@@ -5,10 +5,60 @@ class DashboardController < ApplicationController
   def index
     @participants_per_edition = participants_per_edition
     @implication_level = implication_level
+    @average_points_per_edition = average_points_per_edition
+    @first_five_places = first_places
   end
 
   private
 
+  def first_places
+    Edition.not_deleted.includes(:participations, :participants).each_with_object([]) do |edition, data|
+      data << compute_values(edition)
+    end
+  end
+
+  def first_five_points(edition)
+    edition.participations.first(5).map(&:total_points)
+  end
+
+  def first_five_participants(edition)
+    edition.participants.first(5).map(&:username)
+  end
+
+  def compute_values(edition)
+    points = first_five_points(edition)
+    participants = first_five_participants(edition)
+    {
+      year: edition.month_and_year,
+      first_place_points: points[0],
+      second_place_points: points[1],
+      third_place_points: points[2],
+      fourth_place_points: points[3],
+      fifth_place_points: points[4],
+      first_place_username: participants[0],
+      second_place_username: participants[1],
+      third_place_username: participants[2],
+      fourth_place_username: participants[3],
+      fifth_place_username: participants[4]
```

Comment:
We might group the keys, `x_place_points` and `x_place_username`. Also, considering that the username is not always self-explanatory, we might use the `full_name` method from the `Participant` model. We can discuss it together.